### ISSUE TITLE
PMID:23836910

### ISSUE THREAD

**Issue by Antonialock** [pombase/fypo#2453]
decreased sodium export from cell
normal sodium export from cell 
abolished sodium export from cell 
increased sodium export from cell 

- I think only sod2 is annotated to the sodium export from cell terms so I'll have a look at its other annotations and check if they can all change to this.

also PMID:9622480

**Comment by Antonialock** [comment]
all current annotations to abnormal sodium export can be changed to abnormal sodium export from cell so whichever way is easier for you (rename or create new terms completely?)

**Comment by mah11** [comment]
actually, it would be really quick & easy to rejig the existing term ...

**Comment by mah11** [comment]
I checked, and all* of the other import and export terms that don't specify an organelle are defined (text and xp) as to/from the cell, and have "into cell/from cell" exact synonyms. So I haven't even renamed the sodium terms; I've just changed the "sodium export from cell" synonyms to exact and changed the GO term in the xps.

*yep, all the FYPO terms about moving anything except sodium in/out. I guess I added the FYPO terms before GO added its "sodium ion export from cell" term.
